cluster/src/main/java/com/hartwig/pipeline/tertiary/protect/ProtectCommand.java

```diff
@@ -0,0 +1,52 @@
+package com.hartwig.pipeline.tertiary.protect;
+
+import java.util.List;
+
+import com.hartwig.pipeline.execution.vm.JavaJarCommand;
+import com.hartwig.pipeline.tools.Versions;
+
+public class ProtectCommand extends JavaJarCommand {
+
+    public ProtectCommand(final String tumorSample, final List<String> primaryTumorDoids, final String outputDir,
+            final String actionabilityDir, final String doidJsonPath, final String germlineReportingTsvPath, final String purplePurityPath,
+            final String purpleQCFilePath, final String purpleDriverCatalogPath, final String purpleSomaticVariantsPath,
+            final String bachelorTsvPath, final String linxFusionTsvPath, final String linxBreakendTsvPath, final String linxDriversTsvPath,
+            final String linxViralInsertionsTsvPath, final String chordPredictionPath) {
```

Comment:
This has to be some sort of record for parameter count.

Could we remove the output directory as it looks like it's a constant?